src/Models/HLWorkItem.php

```
@@ -722,4 +744,43 @@ public static function markQualityFailed(Database $db, int $id, string $error):
             'quality_status' => (string) ($row['quality_status'] ?? 'pending'),
         ];
     }
+
+    /**
+     * @return array{quality_attempts: int, quality_status: string}
+     */
+    public static function markQualityFailedForOrg(Database $db, int $id, int $orgId, string $error): array
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)